bincrafters_conventions/actions/check_for_download_hash.py

```diff
@@ -6,8 +6,8 @@ def check_for_download_hash(main, file):
     recipe = conanfile.read()
     conanfile.close()
 
-    if re.search(r'tools\.get\(.+\)', recipe):
-        if re.search(r'tools\.get\(.+,\s?sha256=.+\)', recipe):
+    if re.search(r'tools\.get\(.+\)', recipe, re.DOTALL):
+        if re.search(r'tools\.get\(.+,\s+sha256=.+\)', recipe, re.DOTALL):
```

Comment:
Would this also falsely think that 

```
tools.get("example.example")
some other code...
some other code...
some other code...
some other code...
some other code...
customFunction(sha="aa")
```

would be fine?

I believe that was my initial concern with a too broad regex

Reply:
I have fixed this, I think.
And added a test for this case.

Please review..

Reply:
It is not working, because it fails for `tools.get("www.url.com/{}".format(self.version), sha256=sha256)`

Reply:
And fixed again. I've switched to using ast.